Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information

```

Comment:
Can you please set up the minimal apis here with health check?

Or are you going to do it in a separate change?